tests/_apis/league_of_legends/test_LeagueApiV4.py

```diff
@@ -113,6 +113,27 @@ def test_by_summoner(self):
 
         assert ret is expected_return
 
+    def test_by_puuid(self):
+        mock_base_api = MagicMock()
+        expected_return = object()
+        mock_base_api.raw_request.return_value = expected_return
```

Comment:
eventually i need to refactor these tests, this setup is dumb.... 

no actual action needed, just commenting to myself